ui/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <!-- we need Bootstrap CSS for reactstrap but this is the only way to obtain it without installing the entire library -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
```

Comment:
Would using this package help?
https://www.npmjs.com/package/bootstrap-css-only

We can make renovatebot group reactstrap & bootstrap-css-only together.

Reply:
I looked into it but it includes all of the bootstrap stylesheets, while we only need one. And its [size](https://bundlephobia.com/result?p=bootstrap-css-only@4.4.1) means we'd lose some of the performance benefits from removing bootstrap and jquery (which are ~45kb minified and gzipped combined). Since the css fetched via cdn will be cached by the browser, I don't think this will cost us anything except for an occasional manual check when reactstrap needs to be updated.